### PR TITLE
BNH-128 tenant filtering styling/refactor

### PR DESCRIPTION
**Pull request by TomHanmer29**
Put the filters into a collapsible menu and added margins

### CODE REVIEW

**Comment by TomHanmer29** [comment]
image
image

**Review by BenRamchandani — DISMISSED** [review]
This might conflict a bit with Claire's #126 FYI.

Should the arrow on the filter button flip around? Also wonder if the other button should be called "Apply" or something so it's different

**Comment by TomHanmer29** [comment]
Yeah these are good points. I've looked at Claire's branch just now and I think the way she's named her refactored methods makes a bit more sense so I will wait for her to merge in first and then reapply my changes as necessary

**Comment by TomHanmer29** [comment]
Yeah these are good points. I've looked at Claire's branch just now and I think the way she's named her refactored methods makes a bit more sense so I will wait for her to merge in first and then reapply my changes as necessary

**Comment by TomHanmer29** [comment]
I also need to add the option to filter by e.g. "no couples", this should give me the chance to do another refactor to make everything a bit neater :)

**Comment by TomHanmer29** [comment]
I have used the same object here for the filters in the property form and filters for the tenant. Not sure if this is necessarily a great idea cause it could make variable names confusing but I think it will make coding for the matching algorithm quite a bit easier.

File: BricksAndHearts.UnitTests/ControllerTests/Admin/AdminControllerTests.cs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+using System;
```
Comment (BenRamchandani) [inline BricksAndHearts.UnitTests/ControllerTests/Admin/AdminControllerTests.cs:1]: Presumably unused!

**Review by BenRamchandani — APPROVED** [review]

**Review by SleepyUser — APPROVED** [review]

**Pull request merged**